cmake/Modules/opm-core-prereqs.cmake

```diff
@@ -37,6 +37,7 @@ set (opm-core_DEPS
 	# DUNE dependency
 	"dune-common"
 	"dune-istl"
+	"opm-common REQUIRED"
```

Comment:
I've got the same error in dune-cornerpoint. Can you check if the fix you made can be applied there as well (also possibly autodiff, polymer, ...)?